app/controllers/JsonSchemaHelper.scala

```diff
@@ -73,8 +74,8 @@ object JsonSchemaHelper extends Logging {
         val validationResult = JsonSchemaHelper.validRequest(schema, jsonBody)
         validationResult match {
             case Some(res) if res.isSuccess => f
-            case Some(res) => Future.successful(BadRequest(res.toString()))
-            case _ => Future.successful(BadRequest("Missing body"))
+            case Some(res) => Future.successful(BadRequest(Json.toJson(ErrorResponse(List(FailureMessage.InvalidJson)))))
```

Comment:
Gonna be really annoying - can we add a logger.info here to log the contents of res.toString since we aren't sending it back now?

Reply:
I would also love to send the errors back, but because the schema has a 160 character limit, don't want to break that